src/pages/design.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react"
```

Comment:
You can delete this page because now we have the storybook

feel free to add a 
`typography.story.tsx`
`images.story.tsx`

To have a visual way to test behaviors